src/guides/wiki/oba-2x-config-deploy-guide.md

```diff
@@ -72,19 +74,21 @@ If you are keen to make changes to the source code of OBA, please see the [Enter
+You can download pre-compiled binaries for OneBusAway from the [Downloads](/downloads) page. If you wish to browse the repository to handpick the version you want to use, make sure to download the *onebusaway-transit-data-federation-webapp*, *onebusaway-api-webapp* and *onebusaway-enterprise-acta-webapp* files that resulted from the same version build. This can be determined based on the name of the directory which reflects the respective version name of the build (e.g. 2.0.0-SNAPSHOT) that each individual jar or war files are located in. **Note:** Some versions of OBA are not compatible with Tomcat 8 and Java 11, you may need to use an older OS and older version of Java.
 
-If you do not care about versions, go for the latest by downloading them with the following commands:
+If you have no preference with versions, go for the latest by downloading them with the following commands:
 
     mkdir /oba
     cd /oba
-    wget https://repo.camsys-apps.com/releases/org/onebusaway/onebusaway-transit-data-federation-builder/2.0.0/onebusaway-transit-data-federation-builder-2.0.0-withAllDependencies.jar
-    wget https://repo.camsys-apps.com/releases/org/onebusaway/onebusaway-transit-data-federation-webapp/2.0.0/onebusaway-transit-data-federation-webapp-2.0.0.war
-    wget https://repo.camsys-apps.com/releases/org/onebusaway/onebusaway-api-webapp/2.0.0/onebusaway-api-webapp-2.0.0.war
-    wget https://repo.camsys-apps.com/releases/org/onebusaway/onebusaway-enterprise-acta-webapp/2.0.0/onebusaway-enterprise-acta-webapp-2.0.0.war
+    wget https://repo.camsys-apps.com/releases/org/onebusaway/onebusaway-transit-data-federation-builder/CURRENTVERS/onebusaway-transit-data-federation-builder-CURRENTVERS-withAllDependencies.jar
+    wget https://repo.camsys-apps.com/releases/org/onebusaway/onebusaway-transit-data-federation-webapp/CURRENTVERS/onebusaway-transit-data-federation-webapp-CURRENTVERS.war
+    wget https://repo.camsys-apps.com/releases/org/onebusaway/onebusaway-api-webapp/CURRENTVERS/onebusaway-api-webapp-CURRENTVERS.war
+    wget https://repo.camsys-apps.com/releases/org/onebusaway/onebusaway-enterprise-acta-webapp/CURRENTVERS/onebusaway-enterprise-acta-webapp-CURRENTVERS.war
+
+In the above command, note **CURRENTVERS**. Replace these with the current version, found at the top of the [Downloads](/downloads) page.
```

Comment:
I was hoping we could actually dynamically substitute in the the value of `oba_version` from site_metadata.yml here, but if that isn't immediately feasible, that's ok.